lib/glimesh/accounts/user.ex

```diff
@@ -10,6 +10,9 @@ defmodule Glimesh.Accounts.User do
   schema "users" do
     field :username, :string
     field :displayname, :string
+    field :pronoun_id, :string
+    field :pronoun_stream, :boolean
+    field :pronoun_profile, :boolean
```

Comment:
For these fields, can we give them a little more clarity? I'm thinking we change `pronoun_id` to `pronoun`, and append `show_` to the `pronoun_stream` and `pronoun_profile`. This way we don't think the `pronoun` field is an integer ID, and we know the stream & profile ones are visual toggles.

Reply:
Changes made as requested. I also made the 2 exs files into one file, rather than across 2 as it was, made sense to have the pronouns stuff together in that way.